releasenotes/notes/google-auth-denylist-e7fea0333c11973c.yaml

```diff
@@ -0,0 +1,4 @@
+---
+other:
+  - |
+    Code Security: Add google auth modules to the IAST denylist.
```

Comment:
Can we make this release note more meaningful to an end user? It's not clear to me what are the implications of adding things to the IAST denylist.

Reply:
Done, PTAL.